app/commands.py

```diff
@@ -228,3 +228,23 @@ def run(self, start_date, end_date):
                 process_end_date.isoformat()
             ))
             send_processing_time_for_start_and_end(process_start_date, process_end_date)
+
+
+class PopulateServiceEmailReplyTo(Command):
+
+    def run(self):
+        services_to_update = """
+            INSERT INTO service_email_reply_to(id, service_id, email_address, is_default, created_at)
+            SELECT '{}', id, reply_to_email_address, true, '{}'
```

Comment:
Have you tried the select statement? not in can be slow, just check it's ok

Reply:
I think it's okay purely because the `service_email_reply_to` will only contain entries for services that have updated their email since #1237, which will only be a couple at most.

Reply:
fair enough